eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/components/soap/SOAPBindingServiceComposite.java

```diff
@@ -220,6 +221,11 @@ public void widgetSelected(final SelectionEvent e) {
         GridData upChxGD = new GridData(GridData.FILL_HORIZONTAL);
         upChxGD.horizontalSpan = 3;
         _unwrappedPayloadCheckbox.setLayoutData(upChxGD);
+        
+        _copyNamespacesCheckbox = createCheckbox(composite, "Copy Namespaces");
+        GridData cnChxGD = new GridData(GridData.FILL_HORIZONTAL);
```

Comment:
I don't see differences in SOAPBindingServiceComposite compared to SOAPBindingReferenceComposite (at least for the code added)
Can't we factorize the code?